3-fleetscope/modules/env_baseline/main.tf

```diff
@@ -40,6 +40,10 @@ resource "google_gke_hub_namespace" "fleet-ns" {
   scope_id           = google_gke_hub_scope.fleet-scope[each.key].scope_id
   scope              = google_gke_hub_scope.fleet-scope[each.key].name
   project            = google_gke_hub_scope.fleet-scope[each.key].project
+
+  labels = {
+    "istio-injection" = "enabled"
```

Comment:
this will enable istio by default on all namespaces

I think there is no problem in doing that, services should still be able to communicate with one another